src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -1956,6 +1961,14 @@ def get_crash_data(crash_data, symbolize_flag=True):
         PYTHON_STACK_FRAME_FUNCTION_REGEX, line, state, group=3):
       continue
 
+  # Only get repo.prop if we have and Android kernel or KASAN crash
+  if environment.platform() in ('ANDROID', 'ANDROID_KERNEL') and (
```

Comment:
I don't think we need to check environment.platform here, just "found_android_kernel_crash or is_kasan" should be enough.

Reply:
The theory here is a non Android KASAN kernel should not get repro

Reply:
Right now, it will do environment.platform() which checks env var on every line parsed in stacktrace, very slow. I think found_android_kernel_crash and is_kasan should indicate android kernel and in other cases should bail out.

Reply:
When in non kasan (just straight kernel) crashes we want the repo.prop as well hence the or.  I can remove the platform check, it just means we will get the repro every time.  We should also probably check if the state.android_kernel_repo  is still none otherwise we'll download it more than once